addons/live-comments/projects-studios.js

```diff
@@ -0,0 +1,158 @@
+const wait = (ms) => new Promise((resolve) => setTimeout(resolve, ms));
```

Comment:
Can't `setTimeout` be used directly? This seems to only be called once in the file.

Reply:
its a nice shorthand for a sleep function